control-plane-operator/controllers/hostedcontrolplane/assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -39,7 +55,12 @@ spec:
             - mountPath: /etc/openshift/kubeconfig
               name: kubeconfig
               readOnly: true
+            - mountPath: /var/payload
+              name: payload
+              readOnly: true
```

Comment:
making this read-only appears to also make it read-only for the init container :(